ArduPlane/altitude.cpp

```diff
@@ -907,3 +937,24 @@ float Plane::get_landing_height(bool &rangefinder_active)
 
     return height;
 }
+
+/*
+  if a terrain location doesn't have the relative_alt flag set
+  then fix the alt and trigger a flow of control error
+ */
+void Plane::fix_terrain_WP(Location &loc, uint32_t linenum)
+{
+    if (loc.terrain_alt && !loc.relative_alt) {
+        AP::internalerror().error(AP_InternalError::error_t::flow_of_control, linenum);
```

Comment:
Probably worth a new enum value.

Reply:
I think flow_of_control is right, we haven't yet had an issue finding the line, and I don't expect these will ever trigger